### DISCUSSION TITLE
Welcome to necesse-docker-server Discussions!

### DISCUSSION THREAD

**Discussion by BrammyS** [BrammyS/necesse-docker-server#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BenjiTecK** [comment]
Dear man,

why aren't you using steamcmd in your docker to get automaticaly the serveur update ?

Br,

Benji

**Reply by BrammyS** [reply]
I didn't want to be dependent on SteamCMD when there is no real image size benefit to using it. 
Would you like to be using SteamCMD instead?

**Reply by BenjiTecK** [reply]
that was a suggestion, i'm often hosting game server for my friends and as steam push the update it's easier to maintain on my point of view cause people have simply to rebuild the docker image with their compose to get the last server update if i'm correct.
and as yesterday there is the 1.0.2 patch & fix the client version where different from the server version.

**Comment by voxpop9** [comment]
Hey, 

I suggest adding the mods volume as default for the necesse docker image. 
It does work as I have mods on the server
